lineapy/api/api.py

```diff
@@ -389,14 +389,17 @@ def create_pipeline(
     return pipeline
 
 
-def get_function(artifact_list, input_parameters=[]) -> Callable:
+def get_function(artifact_list, input_parameters=[], use_cache=[]) -> Callable:
```

Comment:
I don't think cache is the right name here. `use_cache` gives off the impression that this is a binary flag that can be used to speed up the computation of get_function without changing its output, which is not the case here. 

My understanding is this is our use case is to have the artifact names provided in this parameter be the start of the computation. I am thinking of a name like `artifacts_to_get_from_artifact_store` or if we prefer something shorter `artifacts_to_load` ?

Reply:
or maybe `input_artifacts_from_artifact_store` and rename `artifact_list` to `output_artifacts`?